airflow/api_fastapi/app.py

```diff
@@ -112,34 +112,29 @@ def get_auth_manager_cls() -> type[BaseAuthManager]:
     return auth_manager_cls
 
 
-def init_auth_manager() -> BaseAuthManager:
-    """
-    Initialize the auth manager.
-
-    Import the user manager class and instantiate it.
-    """
+def create_auth_manager() -> BaseAuthManager:
+    """Create the auth manager."""
     global auth_manager
     auth_manager_cls = get_auth_manager_cls()
     auth_manager = auth_manager_cls()
+    return auth_manager
+
+
+def init_auth_manager(app: FastAPI | None = None) -> BaseAuthManager:
+    """Initialize the auth manager."""
+    auth_manager = create_auth_manager()
     auth_manager.init()
+
+    auth_manager_fastapi_app = auth_manager.get_fastapi_app()
+    if app and auth_manager_fastapi_app:
```

Comment:
```suggestion
    if app and auth_manager_fastapi_app := auth_manager.get_fastapi_app():
```

nit; feel free to reject this one, definitely stylistic :)